src/mscorlib/src/System/DateTime.cs

```diff
@@ -781,9 +781,9 @@ private static long TimeToTicks(int hour, int minute, int second)
             // and y100 are relative to year 1, not year 0
             bool leapYear = y1 == 3 && (y4 != 24 || y100 == 3);
             int[] days = leapYear? DaysToMonth366: DaysToMonth365;
-            // All months have less than 32 days, so n >> 5 is a good conservative
-            // estimate for the month
-            int m = n >> 5 + 1;
+			// All months have less than 32 days so n >> 5, which is the same as n / 32,
+			// is a good conservative estimate for the month
+			int m = (n >> 5) + 1;
```

Comment:
More importantly, though, doesn't this result in a different value for m in some cases? e.g. if n == 0, m will be 0 in the old code but 1 in the new code.